### PR TITLE
vagrant-service-manager version=1.1.0 release=1

### PR DESCRIPTION
**Pull request by navidshaikh**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
 - CHANGELOG fix and README update for OS support for tests @budhrg
+- Fix #220: Bypass hook if no supported guest/box found @budhrg
+- Issue #212 Updating the CONTRIBUTING page with latest guidelines @hferentschik
 - Fix #188: Name of k8s service not consistent @budhrg
 - Fix #225: service-manager env throws NameError @budhrg
 - Fix #168: Extend --debug flag to show plugin activity @budhrg
+- Fixed help messages for box and status commands
 - Don't set private network for unsupported box @budhrg
 - Convert CONTRIBUTING and README docs to AsciiDoc @bexelbie
+- Fix #235: Unable to access docker daemon from host
```
Comment (brgnepal) [inline CHANGELOG.md]: Few missing user name with commit message

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,18 +1,29 @@
...
 - Fix #172: Implement "start/enable" service command @budhrg
+- Issue #172 Modifying Rake CDK download task to allow downloading latest nightly build @hferentschik
+- Pre-release v1.1.0.beta.1 @navidshaikh
 - Fix #237: README and CONTRIBUTING should make use of Asciidoc's :toc: feature @hferentschik
 - Fix #230: Improve acceptance test run time @hferentschik
 - Fix #214: Update acceptance tests to support Mac OS without installing Libvirt @hferentschik
+- Fix #247: Moved status test into service-operation @hferentschik
+- Issue #211 Adding configuration for CI build @hferentschik
+- Fix #210: Adds docker registry URL in openshift env info
```
Comment (brgnepal) [inline CHANGELOG.md]: Here too. username missing.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,18 +1,29 @@
...
+## v1.1.0 Jun 08, 2016
 - Updated README to make Installation Instructions clearer @bexelbie
 - Fix #195: Adding Cucumber and Aruba based acceptance tests @hferentschik
 - CHANGELOG fix and README update for OS support for tests @budhrg
+- Fix #220: Bypass hook if no supported guest/box found @budhrg
+- Issue #212 Updating the CONTRIBUTING page with latest guidelines @hferentschik
 - Fix #188: Name of k8s service not consistent @budhrg
 - Fix #225: service-manager env throws NameError @budhrg
 - Fix #168: Extend --debug flag to show plugin activity @budhrg
+- Fixed help messages for box and status commands
```
Comment (brgnepal) [inline CHANGELOG.md]: Here too. username missing.

File: vagrant-service-manager.spec [hunk 4]
```diff
@@ -71,23 +71,50 @@ popd
...
+- Bumps verison to 1.1.0
+- Updated README to make Installation Instructions clearer @bexelbie
+- Fix #195: Adding Cucumber and Aruba based acceptance tests @hferentschik
+- CHANGELOG fix and README update for OS support for tests @budhrg
+- Fix #220: Bypass hook if no supported guest/box found @budhrg
+- Issue #212 Updating the CONTRIBUTING page with latest guidelines @hferentschik
+- Fix #188: Name of k8s service not consistent @budhrg
+- Fix #225: service-manager env throws NameError @budhrg
+- Fix #168: Extend --debug flag to show plugin activity @budhrg
+- Fixed help messages for box and status commands
```
Comment (brgnepal) [inline vagrant-service-manager.spec]: Here too and few other lines . username missing.

**Comment by brgnepal** [comment]
LGTM with suggested changes.

**Comment by praveenkumar** [comment]
LGTM.

**Comment by hferentschik** [comment]
I think this is out of date

**Comment by LalatenduMohanty** [comment]
We have too many commits for this release, we should have done at least one release before this release. We should follow release early and release often policy.

**Comment by navidshaikh** [comment]
Merged.

**Pull request closed without merging**